GitHub-Org-Management-Policy.md

```diff
@@ -1,31 +1,23 @@
 # Node.js GitHub Organization Management Policy
 
-The Node.js Foundation Github Organization (https://github.com/nodejs) is
-provided as a development resource by the Node.js Foundation under the
-joint operational direction of the Node.js Technical Steering Committee (TSC)
-and Node.js Foundation Community Committee (CommComm).
+The Node.js Foundation GitHub Organization (https://github.com/nodejs) is
+provided as a development resource by the Node.js Foundation under the direction
+of the Node.js Technical Steering Committee (TSC) and Node.js Foundation
+Community Committee (CommComm).
 
 ## Node.js Admin Repository
 
-The Node.js Admin Repository (nodejs/admin) will serve the central location for
-managing Node.js GitHub Organization administrative activities. Only Node.js
+The Node.js Admin Repository (nodejs/admin) will serve as the central location
+for managing Node.js GitHub Organization administrative activities. Only Node.js
 GitHub Organization owners, TSC members, and Community Committee members will
 have write permissions to the nodejs/admin repository.
 
 ## Organization Roles
 
 ### Owners
 
-The Node.js Foundation Executive Director, Node.js TSC Director, TSC Chair,
-Community Committee Chair, and Node.js Foundation Education Manager shall be
-the only individuals granted Owner permissions within the Node.js GitHub
-Organization. Should, at any point in the future, the Node.js Foundation
-Board establish a Community Committee Director position equivalent to the
-TSC Director, that individual would also be automatically granted Owner
-permissions within the organization.
-
-Upon the completion of the terms of service for each of these individuals,
-their Owner permissions within the GitHub organization shall be removed.
+The TSC Director shall be the only individuals granted Owner permissions within
```

Comment:
`individuals` should be singular

Reply:
Thanks. I removed `Director`. (Missed it the first time.) Should say: The TSC shall be the only individuals granted Owner permissions....

(Note that I'm not saying this is the way it *should* be but it is the way it *is* and there's at least some resistance to changing it. Which reminds me, that I should add a Ref in one of these comments to the issue where this is being discussed.)